ext/RMagick/rmutil.c

```diff
@@ -808,8 +805,7 @@ rm_write_temp_image(Image *image, char *temp_name, size_t temp_name_l)
 void
 rm_delete_temp_image(char *temp_name)
 {
-    GVL_STRUCT_TYPE(DeleteImageRegistry) args = { temp_name+5 };
-    MagickBooleanType okay = (MagickBooleanType)CALL_FUNC_WITHOUT_GVL(GVL_FUNC(DeleteImageRegistry), &args);
+    MagickBooleanType okay = DeleteImageRegistry(temp_name+5);
```

Comment:
`rm_delete_temp_image()` has been used in `Draw_destroy`, `Info_free`, etc.
https://github.com/rmagick/rmagick/blob/1d3857689e2211ab9058c7a9254729f33ef4ff92/ext/RMagick/rmdraw.c#L1308

Seems that it causes thread to deadlock when release GVL on destroying gabaged object using Ractor with Ruby 3.0 only.
I think this behavior is Ruby 3.0's bug...

Reply:
To unify the code, `SetImageRegistry` has also stopped releasing GVL